src/Style/web/BaseStyle.js

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from 'styled-components';
+
+const BaseStyle = createGlobalStyle`
```

Comment:
`BaseStyles`
Also, don't forget to rename the filenames for the all the global style components